src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/logs/logs.component.ts

```diff
@@ -15,6 +15,23 @@ import { Icons } from '~/app/shared/enum/icons.enum';
   styleUrls: ['./logs.component.scss']
 })
 export class LogsComponent implements OnInit, OnDestroy {
+  @Input()
+  hideClusterLogs = false;
+  @Input()
+  hideAuditLogs = false;
+  @Input()
+  hideDaemonLogs = false;
+  @Input()
+  hideNavLinks = false;
+  @Input()
+  hideFilterTools = false;
+  @Input()
+  hideDownloadCopyButton = false;
```

Comment:
There's, I think, an interesting reflection on this... In general I find it easier to deal with positive/affirmative parameters: `showClusterLogs`, `enableFeature`, `doThis`, etc., rather than their counterparts: `hideClusterLogs`, `disableFeature`, `dontDoThis`...

Dealing with negative params leads to a slightly more convoluted logic, and when implementing that you have to always negate those (`*ngIf="!hideClusterLogs` vs. `ngIf="showClusterLogs"`). Not a big issue, but worthy to note from "developer experience" ;-)

Reply:
I liked the poisitivity there so I changed them!